src/Development/IDE/Core/Shake.hs

```diff
@@ -429,7 +417,14 @@ shakeRestart it@IdeState{shakeExtras=ShakeExtras{logger}, ..} systemActs =
         shakeSession
         (\runner -> do
               (stopTime,queue) <- duration (cancelShakeSession runner)
-              logDebug logger $ T.pack $ "Restarting build session (aborting the previous one took " ++ showDuration stopTime ++ ")"
+              res <- shakeDatabaseProfile shakeProfileDir shakeDb
+              let profile = case res of
+                      Just fp ->
+                          let link = case filePathToUri' $ toNormalizedFilePath' fp of
+                                          NormalizedUri _ x -> x
+                          in ", profile saved at " <> T.unpack link
```

Comment:
Why are we packing and unpacking the filepath? Why not just dump fp into the message?

Reply:
This is what the code was doing previously, in an attempt to generate a clickable link. That never worked for me, so I'm happy to remove it.